src/nginxconfig/templates/domain_sections/https.vue

```diff
@@ -188,6 +225,18 @@ THE SOFTWARE.
                 </div>
             </div>
         </div>
+        <template v-if="$props.data.http3.value === true">
```

Comment:
```suggestion

        <template v-if="$props.data.http3.computed">
```
Linebreak + use the computed value so that the notice doesn't show if portuse is enabled but http3 is disabled + don't need to compare to `true`, as true is truthy.